recipes/abseil/all/CMakeLists.txt

```
@@ -8,4 +8,8 @@ if(MSVC)
     add_definitions("-D_HAS_DEPRECATED_RESULT_OF")
 endif()
 
+if(NOT DEFINED CMAKE_SYSTEM_PROCESSOR)
```

```suggestion
if(NOT CMAKE_SYSTEM_PROCESSOR)
```

Actually `CMAKE_SYSTEM_PROCESSOR` is always defined, but empty by default when cross-building, so it's better to just check `NOT CMAKE_SYSTEM_PROCESSOR`.